src/test/java/duke/task/ToDoTest.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import task.ToDo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+
+    @Test
+    public void addToDoTest() {
+        ToDo todo = new ToDo("add todo 1");
+        assertEquals("[T][ ] add todo 1", todo.toString());
+    }
+
+    @Test
+    public void saveToDoTest() {
```

Comment:
Try to follow the naming conventions for test methods as shown below:
`featureUnderTest_testScenario_expectedBehavior()`